### DISCUSSION TITLE
I made a chrome extension to directly link my files to the Flat Viewer

### DISCUSSION THREAD

**Discussion by gileadekelvin** [githubocto/flat-action#57] [Flat in Action]
Hi! I made a chrome extension to directly link my files to the Flat Viewer. I added a button next to the delete icon.

It's open source! https://github.com/gileadekelvin/flatdata-web-extension

What you guys think?

![image](https://user-images.githubusercontent.com/10724115/130297614-bc98ce8d-7c7a-46eb-a40c-059a4782ee38.png)

**Comment by irealva** [comment]
This is pretty cool! Thanks for sharing, definitely makes it a lot easier to access the Flat Viewer :)

**Comment by declann** [comment]
Simple and sweet, neat work, thanks! :)
